src/ray/stats/stats.h

```diff
@@ -96,14 +96,14 @@ static inline void Init(const TagsType &global_tags,
   StatsConfig::instance().SetHarvestInterval(
       absl::Milliseconds(std::max(RayConfig::instance().metrics_report_interval_ms() / 2,
                                   static_cast<uint64_t>(500))));
-
-  MetricPointExporter::Register(exporter, metrics_report_batch_size);
-  OpenCensusProtoExporter::Register(
-      metrics_agent_port, (*metrics_io_service), "127.0.0.1", worker_id);
   opencensus::stats::StatsExporter::SetInterval(
       StatsConfig::instance().GetReportInterval());
   opencensus::stats::DeltaProducer::Get()->SetHarvestInterval(
       StatsConfig::instance().GetHarvestInterval());
+
+  MetricPointExporter::Register(exporter, metrics_report_batch_size);
```

Comment:
This is the root cause of slow report. Since we reported before setting the interval, the first interval was always the default (10s)